src/app/comment-period/comment-period.component.html

```diff
@@ -68,7 +68,12 @@ <h3 class="mb-1" [innerHTML]="loading ? 'Loading Comments...' : 'Comments'"></h3
                   <tr class="activity-feed__item" *ngFor="let item of commentPeriod.comments | orderBy: {property: column, direction: direction} | paginate: config">
                     <td colspan="2">
                       <div class="title-container">
-                        <h3 class="title-container__title">{{item.author}}</h3>
+                        <h3 class="title-container__title" *ngIf="item.isAnonymous;else notAnon">
```

Comment:
What about using the ```[innerHTML]``` directive?

This code could become something like:
```<h3 class="title-container__title" [innerHTML]="item.isAnonymous ? 'Anonymous' : item.author "```></h3>

This is mostly to avoid having if/else statements connected to html elements that only differ in the content and not in the structure. The code _should_ be cleaner and more legible this way.